tests/queries/0_stateless/03394_dictionary_asynchronous_metrics.sql

```diff
@@ -0,0 +1,47 @@
+-- Tags: no-parallel
+-- Tag no-parallel -- due to sleep calls
```

Comment:
Sleep calls shouldn't block a test from being able to run in parallel. Looking at the test, it's perfectly fine to run in parallel